os_switch.h

```diff
@@ -118,6 +118,8 @@ class OS_Switch : public OS {
 
 	void key(uint32_t p_key, bool p_pressed);
 
+	virtual String get_custom_platform_resource_pack_location() { return String("romfs:/game.pck"); }
```

Comment:
Was this always a thing, or is it specific to Pandemonium?

Reply:
No I added this 2 days ago.

https://github.com/Relintai/pandemonium_engine/commit/fa2a54a01970bf2ff81c56750765042aeed25efb